packages/shared/lib/core/account/actions/createNewAccount.ts

```diff
@@ -0,0 +1,26 @@
+import { localize } from '@core/i18n'
+import { activeProfile, addAccountMetadataToActiveProfile, addAccountToActiveProfile } from '@core/profile'
+import { createStardustAccount, getAccount } from '@core/profile-manager'
+import { get } from 'svelte/store'
+import { IAccountState } from '../interfaces'
+import { buildAccountStateAndMetadata } from './buildAccountStateAndMetadata'
+
+export async function createNewAccount(name?: string, color?: string): Promise<IAccountState> {
+    const { accounts } = get(activeProfile)
+    try {
+        const createdAccount = await createStardustAccount({
+            alias: name || `${localize('general.account')} ${(get(accounts)?.length ?? 0) + 1}`,
+            coinType: 4219,
```

Comment:
Let's use an enum here

Reply:
Shouldn't this coinType be moved to a profileManager?

Reply:
Made it an object literal and key now uses the network protocol